### PR TITLE
Databricks DBR 5.5 LTS compatibility

### PR DESCRIPTION
**Pull request by metasim**
Main benefit of this is that it provides Spark 2.4 compatibility.

### CODE REVIEW

File: pyrasterframes/src/main/python/scene_30_27_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,655 @@
+{
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/scene_30_27_model.ipynb]: Looks like this got committed inadvertently ... please remove.

File: pyrasterframes/src/main/python/setup.py [hunk 2]
```diff
@@ -130,6 +130,26 @@ def initialize_options(self):
...
+requests = 'requests'
+pytest_runner = 'pytest-runner'
+setuptools = 'setuptools>=0.8'
+ipython = 'ipython==6.2.1'
+ipykernel = 'ipykernel==4.8.0'
+pweave = 'Pweave==0.30.3'
+fiona = 'fiona==1.8.6'
+rasterio = 'rasterio>=1.0.0'
+folium = 'folium'
+pytest = 'pytest>4.0.0,<5.0.0'
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/setup.py]: possibly pytest>=4.0.0 ?

**Comment by metasim** [comment]
Build associated with `40d8a35` failed only due to some time limit (it was otherwise running fine):

```console
The job exceeded the maximum time limit for jobs, and has been terminated.
```

**Comment by vpipkt** [comment]
Build output looks good but it is saying it failed. I don't know what that is about. I would say :shipit:

**Pull request merged**